server_config.cc

```diff
@@ -4,54 +4,92 @@
 #include <string>
 
 ServerConfig::ServerConfig(NginxConfig config) {
-  fillOutMap(config);
+  std::vector<std::string> basePath = {};
+  fillOutMap(config, basePath);
+  printPropertiesMap();
 }
 
-ServerConfig::~ServerConfig() {
-  // TODO
+ServerConfig::~ServerConfig() {}
+
+// TODO: is this a terrible function name?
+// Helper function to concatenate multi-word config parameter names
```

Comment:
Could you add a comment that gives an example portion of a config file that shows how this helper is useful? I understand the overall logic of the new `fillOutMap()`, but not how this helper is used.